pom.xml

```diff
@@ -51,7 +51,7 @@
       <maven.compiler.target>1.8</maven.compiler.target>
       <!-- Versions -->
       <version.cdi2>2.0</version.cdi2>
-      <version.weld3>3.0.2.Final</version.weld3>
+      <version.weld3>3.0.4-SNAPSHOT</version.weld3>
```

Comment:
Why snapshot dep?

Reply:
Oops, a mistake of course. I don't even remember I changed this ;-)